modules/nf-core/purecn/coverage/main.nf

```diff
@@ -1,6 +1,7 @@
 process PURECN_COVERAGE {
     tag "$meta.id"
     label 'process_low'
+    stageInMode "link"
```

Comment:
This might not be optimal, but the way PureCN handles paths using `normalizePath` (https://github.com/lima1/PureCN/blob/ffcff2284fab21ea314849514bdac8edad3f3b1c/inst/extdata/Coverage.R#L125) breaks the way Nextflow handles staging files: the BAM and BAI files end up in different directories and the whole process fails. Help welcome here.

Reply:
Isn't the default of `stageInMode` already `link`?

Reply:
Ah my bad, `symlink` is the default

Reply:
I don't think there is another way to solve this if the tool is written this way :/ You should maybe add this to the description of the tool, because some institutional pipeline overwrite the `stageInMode` in the config. Maybe add some sort of error message to the tool which checks the `stageInMode` and returns the appropiate error so it will be easily fixed by the user (I'm not sure you can access this value though)

Reply:
Should have been fixed. Added the warning on the description and the code for handling the case when stageInMode is not on 'link'. Just please check if it's fine the way it's handled :)